### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 10

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#282] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152319225-2c4456e4-7de2-4ce2-887f-977514e5c99d.png)

**Comment by LuginiAndrea** [comment]
1. Vero: difatti, se valesse sia $aRb$ che $bRa$ avremmo che $b = a$ per anti-simmetria di $R$, quindi $\nexists aR^*b$. In quanto $aR^*b$ esiste, allora non può valere $bRa$, ne tantomeno $bR^*a$
2. Vero o Falso (dipende da come interpretiamo): 
Se la leggiamo sotto il profilo che se vale una allora le altre due non valgono allora si, è vera: 
$b = a$ invalida ognuna delle altre due, $aR^*b$ invalida $bR^*a$, in quanto implicherebbe che $aRb \wedge bRa \wedge b \neq a$, impossibile in quanto per anti-simmetria verrebbe $b = a$. Ovviamente vale anche nell'altro senso. Ne consegue che se è valida una delle tre non sono valide le altre 2.
Se invece la leggiamo sotto il profilo che una ed una sola delle tre deve sempre valere allora no: in quanto $R$ parziale, $\exists a, b \in A \textrm{ t.c. } a \neq b \wedge \nexists aRb \wedge \nexists bRa \implies \nexists aR^*b \wedge \nexists bR^*a$ 
Il che implica che potrebbe semplicemente non sussistere alcuna relazione fra i due elementi
3. Vero: basti considerare che $aR^*b$ implica che $aRb$, e che per transitività $aRb \wedge bRc \Longrightarrow aRc$. Inoltre, in quanto $b \neq a$, non può sussistere nemmeno il caso in cui $c = a$, in quanto implicherebbe $b = a$ per anti-simmetria.
4. Vero: $bR^*c$ implica $bRc$ e $b \neq c$, $aRb \wedge bRc \Longrightarrow aRc$, e non può sussitere il caso $a = c$ in quanto implicherebbe $b = c$ per anti-simmetria
